datafusion-examples/examples/dataframe.rs

```diff
@@ -59,7 +59,8 @@ use tempfile::tempdir;
 #[tokio::main]
 async fn main() -> Result<()> {
     // The SessionContext is the main high level API for interacting with DataFusion
-    let ctx = SessionContext::new();
+    let config = SessionConfig::new().with_parquet_force_view_metadata(false);
```

Comment:
Same here, we need to make the datatype consistent for insert into case.

Reply:
I think this is because SQL `VARCHAR` is being mapped to `Utf8`